src/App.tsx

```diff
@@ -1,61 +1,26 @@
 import './App.scss';
+import { useCallback, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { filteredTodos, getMaxId } from './services/todos';
+import { Todo } from './types/Todo';
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(filteredTodos);
+  const maxId: number = getMaxId(todos) + 1;
+
+  const getPreparedTodos = useCallback((todo: Todo) => {
+    setTodos(currentTodos => currentTodos.concat(todo));
```

Comment:
```suggestion
    setTodos(currentTodos => [...currentTodos, todo]);
```
this looks a bit cleaner